BridgeAppSDK/SBABridgeManager.m

```diff
@@ -207,7 +213,9 @@ + (void)resendEmailVerification:(NSString*)emailAddress
 }
 
 + (NSURLSessionTask *)loadSurvey:(SBBSurveyReference *)surveyReference completion:(SBABridgeManagerCompletionBlock)completionBlock {
-    return [SBBComponent(SBBSurveyManager) getSurveyByRef:surveyReference.href completion:^(id survey, NSError *error) {
+    return [SBBComponent(SBBSurveyManager) getSurveyByRef:surveyReference.href
+                                            cachingPolicy:SBBCachingPolicyFallBackToCached
```

Comment:
I think we want to stick with the default caching policy for surveys (check cache first) here. If an updated version of a survey is published and the schedule is updated to use it, the href will point to the new version, and it won't be found in the cache if it hasn't been loaded yet so SBBSurveyManager will go fetch it from the server. Existing, published versions of surveys don't ever change so once cached, there's no reason to hit the server looking for them ever again.